pkg/admissioncontroller/webhooks/admission/auditpolicy/admission.go

```diff
@@ -229,6 +217,22 @@ func (h *handler) admitConfigMap(ctx context.Context, request admission.Request)
 	return acadmission.Allowed("configmap change is valid")
 }
 
+func validateAuditPolicySemantics(auditPolicy string) (schemaVersion *schema.GroupVersionKind, errCode int32, err error) {
+	auditPolicyObj, schemaVersion, err := policyDecoder.Decode([]byte(auditPolicy), nil, nil)
+	if err != nil {
+		return nil, http.StatusUnprocessableEntity, fmt.Errorf("failed to decode the provided audit policy err=%v", err)
```

Comment:
Nit: can you make the returned errors consistent?
```suggestion
return nil, http.StatusUnprocessableEntity, fmt.Errorf("failed to decode the provided audit policy: %v", err)
```